django-server/server/api/serializers.py

```diff
@@ -26,9 +26,11 @@ class Meta:
 
 
 class CameraSerializer(serializers.ModelSerializer):
+    name = serializers.RegexField('^[a-zA-Z0-9-_]+$')
+
     class Meta:
         model = Camera
-        fields = ['id', 'state', 'cafeteria']
+        fields = ['id', 'state', 'name', 'cafeteria']
```

Comment:
Could you change order to put name second?

Reply:
> It's okay, Django generates this file and changes lines numbers to match existing code.

Thank's for explanation